metrics/web/test/dashboard/presentation/state/project_metrics_notifier_test.dart

```diff
@@ -1038,6 +1069,36 @@ void main() {
       },
     );
 
+    test(
+      ".setProjectGroups() update the selected project group to the group with an id equals to the page parameters project group id",
+      () {
+        projectMetricsNotifier.handlePageParameters(parameters);
+
+        projectMetricsNotifier.setProjectGroups(projectGroups);
+
+        final actualProjectGroup = projectMetricsNotifier.selectedProjectGroup;
+
+        expect(actualProjectGroup.id, equals(projectGroupId));
+      },
+    );
+
+    test(
+      ".setProjectGroups() update page parameters using the selected project group",
+      () {
+        projectMetricsNotifier.setProjectGroups(projectGroups);
+        projectMetricsNotifier.selectProjectGroup(projectGroupId);
+
+        final selectedProjectGroup =
+            projectMetricsNotifier.selectedProjectGroup;
+
+        projectMetricsNotifier.setProjectGroups(projectGroups);
```

Comment:
✅ 
I'm not sure this test is correct since if we won't call this method here it will pass anyway.